### DISCUSSION TITLE
question on how to find this thing

### DISCUSSION THREAD

**Discussion by baka-aho** [prasanthrangan/hyprdots#1472] [Q&A] [answered]
Got a curiosity and i have taken some liking to the terminal here, I would like to know how to enable it when i open kitty and also edit them, please answer me if you know
![240511_00h26m26s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/136156874/69ee99bd-44f7-4b58-923c-1d9e62cf1b61)

These are the terminals
![240511_00h45m24s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/136156874/a74802b6-5ad6-443f-bc25-11ad491cc40f)
![240511_00h46m53s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/136156874/8688b742-0a44-4a9b-9598-7a541be541e4)

**Comment by dieBakterie** [comment]
The Terminal is kitty and that's a custom bash script, it's located in `.local/share/bin/` and it's called systemupdate.sh, you can run it in the terminal by `> systemupdate.sh up`. It's fastfetch for what you see there with zsh + oh my zsh + p10k pokemon theme.

**Reply by baka-aho** [reply]
Oh right!! but it appears to open a new window + i want to remove some of the characters just the customization without it doing any installation. What do I need to do for that?

**Comment by kRHYME7** [comment]
```
command="
 fastfetch
 "
    kitty --title systemupdate sh -c "${command}"
  ```

 
   
add your commands inside command="...."  better if script.sh

**Reply by baka-aho** [reply]
guess you misunderstood a little, i don't want to execute any codes while running that script but rather i want to customize the info above and remove some of the images. also i think only using 
```bash
command="
 fastfetch
 "
    kitty --title systemupdate sh -c "${command}"
```
doesnt let me open the terminal

**Reply by kRHYME7** [reply, marked as answer]
fastfetch? 


` ~/.config/fastfetch/config.jsonc ` edit that ?

**Reply by baka-aho** [reply]
ohh i dont think i ever downloaded fastfetch, that helps thanks a lot

**Reply by kRHYME7** [reply]
We abandoned neofetch recently
